gsk4/Gir.toml

```diff
@@ -172,6 +172,7 @@ status = "generate"
 [[object]]
 name = "Gsk.Transform"
 status = "generate"
+trust_return_value_nullability = false
```

Comment:
Can you fix the affected functions instead? :) I don't think all of them can return `NULL`, or even most of them not.

Reply:
I can look into it, but not before tomorrow's release :)

None of them are annotated, not even `gsk_transform_translate` or `gsk_transform_invert` so need to do more research and ask people that know how Transforms are implemented.

Reply:
The only one that can return `NULL` is `gsk_transform_invert` and that is annotated accordingly already.